src/status/libstatus/accounts.nim

```diff
@@ -336,3 +361,22 @@ proc deriveAccounts*(accountId: string): MultiAccounts =
 
 proc logout*(): StatusGoError =
   result = Json.decode($nim_status.logout(), StatusGoError)
+
+proc storeIdentityImage*(keyUID: string, imagePath: string, aX, aY, bX, bY: int): IdentityImage =
+  let response = callPrivateRPC("multiaccounts_storeIdentityImage", %* [keyUID, imagePath, aX, aY, bX, bY]).parseJson
```

Comment:
No error handling. Is it possible that this rpc method fails?

Reply:
I `try/catch` it in the view. It was simpler that way since I want this function to return the IdentityImage, but also get a real error message when it fails. 
Let me know if there is a better way to do it. Maybe with a Tuple?